### PR TITLE
Nested udfs error in redshift

### PR DESCRIPTION
**Pull request by alex-graciano**
This refactor saves us from the extra nesting that produces the Redshift error in the `transformations` module:

```
SQL Error [500310] [XX000]: [Amazon](500310) Invalid operation: Nested UDFs are not supported.;
```

https://app.shortcut.com/cartoteam/story/198004/nested-udfs-error-in-redshift

### CODE REVIEW

File: modules/transformations/redshift/lib/helper.py [hunk 1]
```diff
@@ -103,3 +105,45 @@ def calculate_final_bearing(start, end):
     bear = bearing(end, start)
     bear = (bear + 180) % 360
     return bear
+
+
+def geom_from_geojson(geom):
```
Comment (vdelacruzb) [inline modules/transformations/redshift/lib/helper.py:110]: I would rename this functions to avoid confusions `geom_from_geojson`-> `wkt_from_geojson`.

**Review by vdelacruzb — COMMENTED** [review]
I would do the rename that I mentioned and then should be good to go.
Makes me worried that from other modules, the same nesting bug could still be happening as we are not doing stuffs internally but calling `__ST_GEOMFROMGEOJSON`, hope the oficial function arrives soon.

**Comment by alex-graciano** [comment]
> __ST_GEOMFROMGEOJSON

**Review by vdelacruzb — APPROVED** [review]
LGTM

**Review by Jesus89 — COMMENTED** [review]
Could you add a test to verify this is fixed? Thanks

**Comment by alex-graciano** [comment]
I was working on one but I cannot imagine why the Python connector was not let me `CREATE TABLE... AS` from a query. I'll take a look

**Review by Jesus89 — APPROVED** [review]

**Pull request merged**